packages/esm-patient-registration-app/src/offline.resources.ts

```diff
@@ -7,16 +7,16 @@ import { PatientIdentifierType, FetchedPatientIdentifierType } from './patient-r
 import { cacheForOfflineHeaders } from './constants';
 
 export interface Resources {
-  addressTemplate: any;
-  currentSession: Session;
+  addressTemplate: undefined | any;
+  currentSession: Session | undefined;
   relationshipTypes: any;
-  identifierTypes: Array<PatientIdentifierType>;
+  identifierTypes: Array<PatientIdentifierType> | undefined;
```

Comment:
Nitpicky, I know, but I'd prefer to use a `?` instead of the `| undefined` here. For the `any` types it's not really required anyway (though it also doesn't hurt to add it, the `any` could become more concrete eventually and then having the `undefined` type in place already is a good thing).